core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -111,7 +111,7 @@ func (f *FwdMgr) filterName(addr common.Address) string {
 	return evmlogpoller.FilterName("ForwarderManager AuthorizedSendersChanged", addr.String())
 }
 
-func (f *FwdMgr) GetForwarderForEOA(addr common.Address) (forwarder common.Address, err error) {
+func (f *FwdMgr) GetForwarderFor(addr common.Address) (forwarder common.Address, err error) {
```

Comment:
Changed the name because not all chains distinguish between EOAs and Contracts (Starknet for example). On Starknet you always issue transactions from contracts / an account is tied to a contract. So even though you have a private key to control your account, you would have forwarders for the contract address it is tied to.